src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -144,13 +152,15 @@ the main body of the message as well as a quote.
 								{{ action.label }}
 							</ActionButton>
 						</template>
-						<ActionButton
-							v-if="isDeleteable"
-							icon="icon-delete"
-							:close-after-click="true"
-							@click.stop="handleDelete">
-							{{ t('spreed', 'Delete') }}
-						</ActionButton>
+						<template v-if="isDeleteable">
+							<ActionSeparator />
```

Comment:
```suggestion
							<ActionSeparator v-if="isReplyable || messageActions.length > 0" />
```

Otherwise you might start with a separator at some point :P